Wikipedia/Code/AboutViewController.m

```diff
@@ -59,6 +59,15 @@ - (void)wmf_setTextFontSize {
     [self evaluateJavaScript:fontSizeJS completionHandler:nil];
 }
 
+- (void)wmf_setTextFontColor:(WMFTheme *)theme {
+    NSString *fontColorJS = [NSString stringWithFormat:@"document.body.style.color = '#%@';\
+                             document.styleSheets[0].rules[2].style.color = '#%@';\
```

Comment:
@nambatee 
I think I got some JS accessing the selectors by name - ok if I push a commit?